src/AbstractContainerCommandLoader.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/**
+ * @see       https://github.com/laminas/laminas-cli for the canonical source repository
+ * @copyright https://github.com/laminas/laminas-cli/blob/master/COPYRIGHT.md
+ * @license   https://github.com/laminas/laminas-cli/blob/master/LICENSE.md New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Laminas\Cli;
+
+use Laminas\Cli\Exception\ConfigurationException;
+use Psr\Container\ContainerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\CommandLoader\CommandLoaderInterface;
+
+use function array_keys;
+use function class_exists;
+
+/**
+ * @internal
+ */
+abstract class AbstractContainerCommandLoader implements CommandLoaderInterface
+{
+    /** @var ContainerInterface */
+    private $container;
+
+    /** @var string[] */
+    private $commandMap;
+
+    public function __construct(ContainerInterface $container, array $commandMap)
```

Comment:
```suggestion
    final public function __construct(ContainerInterface $container, array $commandMap)
```

You'll need to make this `final` if you want to ensure that the properties are always set and available in extending classes.